From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Tue, 28 Jul 2020 06:11:52 +1200
Subject: Remove commented-out Debian package "libmono-system2.0-cil" for C# in
 chroot

It's out of date. The current version is 4.0, but there is no need to
install it explicitly because "mono-runtime" depends on it indirectly.
---
 misc-tools/dj_make_chroot.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc-tools/dj_make_chroot.in b/misc-tools/dj_make_chroot.in
index f7cf81c48..eefe818ac 100755
--- a/misc-tools/dj_make_chroot.in
+++ b/misc-tools/dj_make_chroot.in
@@ -134,7 +134,7 @@ INCLUDEDEBS="ca-certificates"
 
 # Packages to install after upgrade (space separated):
 INSTALLDEBS="default-jre-headless pypy python3 locales"
-# For C# support add: mono-runtime libmono-system2.0-cil
+# For C# support add: mono-runtime
 # However running mono within chroot still gives errors...
 
 # Packages to remove after upgrade (space separated):
-- 
2.20.1

